ambry-commons/src/test/java/com.github.ambry.commons/ByteBufferReadableStreamChannelTest.java

```diff
@@ -151,68 +170,6 @@ public void idempotentOperationsTest()
     assertFalse("ByteBufferReadableStreamChannel is not closed", byteBufferReadableStreamChannel.isOpen());
   }
 
-  /**
-   * {@link ByteBufferReadableStreamChannel} has no restrictions on when and what algorithms can be used to generate
-   * the digest. This function tests the correctness of such support.
-   * @throws Exception
-   */
-  @Test
-  public void changingAlgorithmsTest()
```

Comment:
Changing algorithms was a special support of `ByteBufferReadableStreamChannel`. Not all `ReadableStreamChannel` impls (like `NettyRequest`) can support this.